src/biosHandler.cpp

```diff
@@ -8,7 +8,11 @@
 
 namespace vpd
 {
-void BiosHandler::checkAndListenPldmService()
+// Template declaration to define APIs.
```

Comment:
is file name should be "bios_handler.cpp" ?

Reply:
I had this question earlier too , just was thinking if we should fix file names to be as a single string without the underscore or have File names with underscore between the words in the name?